packages/@n8n/api-types/src/dto/common.ts

```diff
@@ -0,0 +1,3 @@
+import { z } from 'zod';
+
+export const booleanLiteral = z.union([z.literal('true'), z.literal('false')]);
```

Comment:
[we have a new folder](https://github.com/n8n-io/n8n/tree/master/packages/%40n8n/api-types/src/schemas) for keeping schema objects like these now. perhaps we should move this file to `src/schemas/booleanLiteral.ts` ?
or maybe this could be
```typescript
const booleanFromString = z.enum(['true', 'false']).transform((val) => val === 'true');
```